documentation/modules/exploit/linux/http/dell_idrac_upload_exec.md

```diff
@@ -0,0 +1,85 @@
+## Description  
+
+Dell IDRAC servers prior to firmware version 2.52.52.52 are vulnerable to an unauthenticated remote code execution vulnerability, CVE-2018-1207. This exploit makes use of CGI environment variables to upload a reverse shell and hook LD_PRELOAD to obtain a connection. This process has the potential to fork bomb the system, so this module safely unsets the LD_PRELOAD variable on connection to the reverse shell. The servers run on the SH4 architecture, which requires cross-compiling with the gcc-8-sh4-linux-gnu package. Only a reverse shell is supported at this time.
+
+## Vulnerable Application  
+
+Dell IDRAC 7/8
+
+
+## Vulnerable Application Installation Setup.
+
+Generation 12 and 13 Dell PowerEdge servers are vulnerable. 
+
+https://en.wikipedia.org/wiki/List_of_Dell_PowerEdge_Servers
+
+## Verification Steps  
+
+  1. Identify an IDRAC 7 or 8 appliance
+  2. Start msfconsole
+  3. Do: ```use exploit/linux/http/dell_idrac_upload_exec```
+  4. Do: ``set rhost <ip>``
+  5. Do: ``set rport <80/443>``
+  6. Do: ``check``
```

Comment:
You don't _need_ to do check in order for the exploit to succeed.

Reply:
Should be resolved by 3a3cfc2